Automattic/jetpack
# Subscribe Block: address front-end rendering issues

Follow-up to #22694

This fixes a couple issues with how this block renders on the front-end.

- Ensure the right selectors are targeted for style variations.
- Use box-sizing and white space wrapping to prevent overflows.

![image (55)](https://user-images.githubusercontent.com/548849/154707164-efd4c6c8-f022-47a7-9d6d-75c0a29434bf.png)

Closes #20162.

#### Testing instructions:

* Go to create a new post.
* Insert the Subscribe block.
* Try different combinations of styles; publish and check rendering on the front-end.

#### Does this pull request change what data or activity we track or use?

No
